BackupPC_report

```diff
@@ -159,30 +161,45 @@ for my $host ( @ARGV ? @ARGV : @hosts ) {
 # Supress report if no warnings
 !$opt_s || $warnOutdated || $warnErr || $warnActive || $warnPartial || exit 0;
 
-# Print report header
-print POSIX::strftime( '%Y-%m-%d %H:%M', localtime(time) ), "       ",
-  scalar @hosts, " host(s) configured ($hostsDisabled disabled)\n\n";
-print "*WARNING* Some backups with ERRORs!", "\n\n" if $warnErr;
-print "*WARNING* Most recent backups for some hosts are older than expected!", "\n\n"
-  if $warnOutdated;
-print "*WARNING* There are in-progress backups!", "\n\n" if $warnActive;
-print "*WARNING* There are partial backups!",     "\n\n" if $warnPartial;
-
-# Print BackupPC URL
-printf "\nURL: $bpc->{Conf}{CgiURL}\n\n";
-
-# Print table header
-printf "%4s %6s %-${hostColWidth}s %-16s %6s %3s %6s | %4s %5s %4s %3s | %6s\n",
-  "warn", "backup", "host name", "date/time", "size", "", "durat.",
-  "xfer", "Share", "File", "tar", "age";
-printf
-  "%4s %6s %-${hostColWidth}s %-16s %6s %3s %6s | %-4s %-5s %-4s %3s | %6s\n",
-  "flag", "number", "", "", "(MB)", "lvl", "(min)",
-  "Err", "Err", "Err", "Err", "(days)";
-print "-" x ( $hostColWidth + 78 ), "\n";
-
-# Print the rest of the table
-map { printf "%4s [%4d] %-${hostColWidth}s %-16s %6s %3d %1s%5s | %4s %5s %4s %3s | %6d\n", @$_; } @Report
+if ( $opt_f =~ /report/i ) {
+
+    # Print report header
+    print POSIX::strftime( '%Y-%m-%d %H:%M', localtime(time) ), "       ",
+      scalar @hosts, " host(s) configured ($hostsDisabled disabled)\n\n";
+    print "*WARNING* Some backups with ERRORs!",                                   "\n\n" if $warnErr;
+    print "*WARNING* Most recent backups for some hosts are older than expected!", "\n\n"
+      if $warnOutdated;
+    print "*WARNING* There are in-progress backups!", "\n\n" if $warnActive;
+    print "*WARNING* There are partial backups!",     "\n\n" if $warnPartial;
+
+    # Print BackupPC URL
+    printf "\nURL: $bpc->{Conf}{CgiURL}\n\n";
+
+    # Print table header
+    printf "%4s %6s %-${hostColWidth}s %-16s %6s %3s %6s | %4s %5s %4s %3s | %6s\n",
+      "warn", "backup", "host name", "date/time", "size", "", "durat.",
+      "xfer", "Share", "File", "tar", "age";
+    printf
+      "%4s %6s %-${hostColWidth}s %-16s %6s %3s %6s | %-4s %-5s %-4s %3s | %6s\n",
+      "flag", "number", "", "", "(MB)", "lvl", "(min)",
+      "Err", "Err", "Err", "Err", "(days)";
+    print "-" x ( $hostColWidth + 78 ), "\n";
+
+    # Print the rest of the table
+    map { printf "%4s [%4d] %-${hostColWidth}s %-16s %6s %3d %1s%5s | %4s %5s %4s %3s | %6d\n", @$_; } @Report;
+}
+elsif ( $opt_f =~ /csv/i ) {
```

Comment:
What if we move `csv` block before  `report` and for `report` use an else-block without extra condition as `report` is the default?

Reply:
Also a way, my thought was just to keep clear which format is processed by this section. I'm also fine with the else-block as default for the report option